src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -99,7 +102,15 @@ def __init__(
 
         self._connections = connections
         self._storage = storage
+
+        # process timeout parameters and pass them to create function of executor proxy
+        self._batch_timeout_sec = (
+            batch_timeout_sec if batch_timeout_sec else get_int_env_var("PF_BATCH_TIMEOUT_SEC", None)
+        )
+        self._line_timeout_sec = get_int_env_var("PF_LINE_TIMEOUT_SEC", LINE_TIMEOUT_SEC)
+        kwargs.update({"line_timeout_sec": self._line_timeout_sec})
```

Comment:
We may add some comment why we add this to kwargs?

Reply:
Removed.